polyfill/platform/ARKitWrapper.js

```diff
@@ -72,21 +70,12 @@ export default class ARKitWrapper extends EventHandlerBase {
 	static GetOrCreate(options = null){
 		if(typeof ARKitWrapper.GLOBAL_INSTANCE === 'undefined'){
 			ARKitWrapper.GLOBAL_INSTANCE = new ARKitWrapper()
-			options = (options && typeof(options) == 'object') ? options : {
-				ui : {
-					browser: true,
-					points: true,
-					focus: false,
-					rec: true,
-					rec_time: true,
-					mic: false,
-					build: false,
-					plane: false,
-					warnings: true,
-					anchors: false,
-					debug: false
-				}
-			}
+			options = (options && typeof(options) == 'object') ? options : {}
+			let defaultUIOptions = {
+				browser: true
```

Comment:
We should also default the recording UI to true because we want people to easily share photos and videos.